docs/guides/deploy.md

```diff
@@ -0,0 +1,138 @@
+
+### Existing projects
+
+Existing projects have either been scaffolded by Nitric Deploy or have been created using the Nitric CLI.
+
+> These projects _must_ contain a valid nitric.yaml file for them to function correctly. Deploy will not allow projects without it to be imported.
+
+<img alt="add existing project" src="../../assets/img/guides/deploy/add-project-existing.jpg" height="337.5" />
+
+### Scaffold a new project
+
+Our template projects will give you a starting point with a basic hello world REST API.
+
+> Your new project will be scaffolded in the repository which you select.
+
+<img alt="add new project" src="../../assets/img/guides/deploy/add-project-new.jpg" height="337.5" />
+
+In both of the scenarios you'll need to choose a production branch. This is the branch which Nitric Deploy will watch for any push events and trigger a deployment.
+
+You'll also need to specify which directory the Nitric source code is in - by default this would be './' but could be a few levels deeper if you are working in an existing monorepo.
+
+<br/>
+
+## Configure environment variables
+
+If your project requires variables, they can be added in project settings.
+
+> Changes to variables will require a redeploy, which can be triggered from the projects home.
+
+<img alt="...." src="../../assets/img/guides/deploy/new-environment-variable.jpg" height="337.5" />
+
+<br/>
+
+## Create an environment
+
+An environment represents a collection of deployments targeted at your configured AWS account. You can have one or more environments if you'd like to deploy to different environment stages (e.g. dev, staging, prod), regions, or use different credentials.
+
+> Once you've created an environment, Nitric Deploy will automatically initiate your first deployment to the cloud.
+
+<img alt="...." src="../../assets/img/guides/deploy/new-environment.jpg" height="337.5" />
+
+<img alt="...." src="../../assets/img/guides/deploy/new-environment-2.jpg" height="337.5" />
+
+<br/>
+
+## Deploy your project
+
+Your project will automatically deploy when you create your environment for the first time. It will also deploy each time you push to the branch which was configured when you created your project.
+
+Once you see the deployment tick over to success with a 'tick' you'll also notice that your preview / production URL is displayed.
+
+You can use this to access your deployment, if you scaffolded a template project then you can test the API out with the following command.
+
+```bash
+curl https://{url}/hello/user
+```
+
+> You can change the branch by going to your project settings.
+
+<img alt="Create and deploy" src="../../assets/img/guides/deploy/deploy-2.jpg" height="337.5" />
+
+<br/>
+
+## Check out the logs
+
+By clicking on 'view latest build' we can drill down into the details and retrieve the deployment log file.
+
+<img alt="Deployment logs" src="../../assets/img/guides/deploy/deploy-logs.jpg" height="337.5" />
+
```

Comment:
Add a wrap-up sentence... something like this?

"That's it! With just a few easy clicks, you've deployed a project to your cloud and don't have to manage environments or infrastructure state."